samples/sensor/amg88xx/README.rst

```diff
@@ -0,0 +1,69 @@
+.. _amg88xx:
+
+AMG88XX Infrared Array Sensor
+#############################
+
+Overview
+********
+
+Sample application that periodically reads 8x8 temperature array from
+the AMG88XX sensor. Sample can also be configured to be triggered
+when the upper or lower threshold has been reached.
+
+Requirements
+************
+
+Panasonic AMG88XX is an 8x8 (64) pixel infrared array sensor.
+
+The AMG88XX is available only in a SMD design. There are a Adafruit
+breakout board and an Panasonic Evalutation Kit which are easier to handle.
+
+- `AMG88xx Evaluation Kit`_
+- `Adafruit AMG8833 8x8 Thermal Camera Sensor`_
+
+We used the Evalutation Kit mounted on FRDM-K64F board.
+On the Evalutation Kit, all jumpers except the J11 must be removed.
+
+This sample uses the sensor APIs and the provided driver for the AMG88xx sensor.
+
+Building and Running
+********************
+
+This project outputs sensor data to the console. If you want to test
```

Comment:
How about:

     This sample reads from the sensor and outputs sensor data to the console every second.
     If you want to test the sensor's trigger mode, specify the trigger configuration in the Kconfig
     file and wire the interrupt output from the sensor to your board.  For the FRDM-K64F we tested,
     pin PTC2 must be muxed as GPIO.

I'm not sure of the technical details here, but you should clearly state how to configure and hook up the sensor.